v_2_0_0/master_template.go

```diff
@@ -1508,11 +1300,7 @@ write_files:
        calico-node-sa.yaml\
        calico-kube-controllers-sa.yaml\
        calico-ds.yaml\
-       calico-kube-controllers.yaml\
-       calico-node-controller-sa.yaml\
-       calico-node-controller.yaml\
-       calico-ipip-pinger-sa.yaml\
-       calico-ipip-pinger.yaml"
+       calico-kube-controllers.yaml"
```

Comment:
i think we should add commands for deleting the old removed resources

Reply:
This makes sense.